### ISSUE TITLE
How to add a Cross-browser web extension

### ISSUE THREAD

**Issue by giSe98** [midori-browser/core#461]
I'm trying to add a simple extension but it is not seen by the browser (I added the extension both as crt and as a folder). 
Would anyone know how to show me the steps to take to add the extension and possibly also let me have an example of a working extension?
[hello-world.zip](https://github.com/midori-browser/core/files/9831138/hello-world.zip)

**Comment by ghost** [comment]
Looking at my system extensions are found in `~/.local/share/midori/extensions/` and at least the default ones are all .xpi as in Firefox and its forks.

The .xpi file extension format is actually a .zip. The _hello-world_ link actually points to a RAR archive, despite its .zip file extension. To install it you'll have to extract the contents and then repack it as a true zip archive, then change the .zip extension to .xpi.

**Comment by giSe98** [comment]
Ok, but just put the .xpi file in the **extensions** folder ?

**Comment by ghost** [comment]
Yes. After that you open the browser, go to **Menu > Preferences > Extensions** and enable it in the list.
(**Menu** being the "hamburger" button at the top-right of the main window)

**Comment by giSe98** [comment]
It doesn't work. Do you have an email so I can illustrate the problem better?

**Comment by ghost** [comment]
Try [...]
_(removed e-mail address to prevent spam)_
